collect_app/src/main/java/org/odk/collect/android/preferences/utielities/KnownUrlListUtils.java

```diff
@@ -0,0 +1,44 @@
+package org.odk.collect.android.preferences.utielities;
+
+import com.google.gson.Gson;
+import com.google.gson.reflect.TypeToken;
+
+import org.odk.collect.android.R;
+import org.odk.collect.android.application.Collect;
+import org.odk.collect.android.preferences.GeneralSharedPreferences;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class KnownUrlListUtils {
+    static final String KNOWN_URL_LIST = "knownUrlList";
+
+    private KnownUrlListUtils() {
+    }
+
+    public static void addUrlToList(String url) {
```

Comment:
The intended behavior of this method is non-obvious enough that it would be helpful to have a comment that describes its side effects.

Reply:
Ok I added a comment.